### PR TITLE
Do not crash when user has already login with a service account

### LINKED ISSUE #369: Django Cloud Deploy fail to retrieve the credentials when user logged in gcloud with a service account
## Issue running command
django-cloud-deploy cloudify

## Environment information

Django Cloud Deploy: [0.3.0]

Platform: [Linux-4.19.20-1rodete1-amd64-x86_64-with-Debian-rodete-rodete]

Gcloud Version: [238.0.0]

Cloud SQL Proxy Version: [Cloud SQL Proxy: version 1.13; sha ac834ce67862b7c4ed8bdbe86e664572f820c245 built Tue Oct  9 23:39:16 UTC 2018]

Python Version: [3.6.5 (default, Mar 31 2018, 05:34:57)  [GCC 7.3.0]]

Docker Version: [Docker version 18.09.3, build 774a1f4]

## Traceback
```.pytb
Traceback (most recent call last):
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/django_cloud_deploy/django_cloud_deploy.py", line 46, in _cloudify
    cloudify.main(args)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/django_cloud_deploy/cli/cloudify.py", line 150, in main
    prompt_args)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/django_cloud_deploy/cli/prompt.py", line 1408, in prompt
    creds, active_account)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/django_cloud_deploy/cli/prompt.py", line 1354, in _setup_prompts
    project_client = project.ProjectClient.from_credentials(creds)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/django_cloud_deploy/cloudlib/project.py", line 61, in from_credentials
    cache_discovery=False))
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/venv/lib/python3.6/site-packages/googleapiclient/_helpers.py", line 130, in positional_wrapper
    return wrapped(*args, **kwargs)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/venv/lib/python3.6/site-packages/googleapiclient/discovery.py", line 224, in build
    requested_url, discovery_http, cache_discovery, cache, developerKey)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/venv/lib/python3.6/site-packages/googleapiclient/discovery.py", line 274, in _retrieve_discovery_doc
    resp, content = http.request(actual_url)
  File "/usr/local/google/home/wangxf/Desktop/code/django-cloud-deploy/venv/lib/python3.6/site-packages/google_auth_httplib2.py", line 186, in request
    self.credentials.before_request(
AttributeError: 'NoneType' object has no attribute 'before_request'

```

### PR DESCRIPTION
**Pull request by wangxf123456**
Fixes #369

### CODE REVIEW

File: django_cloud_deploy/cli/prompt.py [hunk 1]
```diff
@@ -714,6 +714,11 @@ def prompt(self, console: io.IO, step: str,
             creds = self.auth_client.create_default_credentials()
         else:
             creds = self.auth_client.get_default_credentials()
+            if not creds:
+                console.ask(
+                    ('Warning: Your existing credentials file is broken. Press '
```
Comment (enriquejosepadilla) [inline django_cloud_deploy/cli/prompt.py]: Broken doesnt explain to me what went wrong, anyway we can be more expressive?  is there a way to give them a link with more information?

Comment (wangxf123456) [reply django_cloud_deploy/cli/prompt.py]: I was not able to find the documentation for the correct credentials json file. So I just made the warning message more specific.

File: django_cloud_deploy/cli/prompt.py [hunk 2]
```diff
@@ -714,6 +714,13 @@ def prompt(self, console: io.IO, step: str,
             creds = self.auth_client.create_default_credentials()
         else:
             creds = self.auth_client.get_default_credentials()
+            if not creds:
+                console.ask(
+                    ('Warning: You are using a service account to authenticate '
+                     'gcloud, or your existing credentials file is either '
```
Comment (enriquejosepadilla) [inline django_cloud_deploy/cli/prompt.py]: either xxx? I believe you missed a word.

Comment (wangxf123456) [reply django_cloud_deploy/cli/prompt.py]: My bad. Fixed.

**Pull request merged**